pygpudrive/env/env_torch.py

```diff
@@ -230,13 +230,17 @@ def _set_continuous_action_space(self) -> None:
             action_1 = self.dx.clone().cpu().numpy()
             action_2 = self.dy.clone().cpu().numpy()
             action_3 = self.dyaw.clone().cpu().numpy()
-        else:
+        elif self.config.dynamics_model == "classic":
             self.steer_actions = self.config.steer_actions.to(self.device)
             self.accel_actions = self.config.accel_actions.to(self.device)
             self.head_actions = torch.tensor([0], device=self.device)
             action_1 = self.steer_actions.clone().cpu().numpy()
             action_2 = self.accel_actions.clone().cpu().numpy()
             action_3 = self.head_actions.clone().cpu().numpy()
+        else:
```

Comment:
Is there any action space for which continuous actions is not supported? It's not obvious to me why this would be